packages/gatsby-plugin-manifest/src/__tests__/gatsby-node.js

```diff
@@ -105,6 +108,45 @@ describe(`Test plugin manifest options`, () => {
     expect(contents).toMatchSnapshot()
   })
 
+  it(`correctly works with multiple icon paths`, async () => {
+    fs.existsSync.mockReturnValue(false)
+
+    const size = 48
+
+    const pluginSpecificOptions = {
+      icons: [
+        {
+          src: `icons/icon-48x48.png`,
+          sizes: `${size}x${size}`,
+          type: `image/png`,
+        },
+        {
+          src: `other-icons/icon-48x48.png`,
+          sizes: `${size}x${size}`,
+          type: `image/png`,
+        },
+      ],
+    }
+
+    await onPostBootstrap(apiArgs, {
+      ...manifestOptions,
+      ...pluginSpecificOptions,
+    })
+
+    const firstIconPath = path.join(
+      `public`,
+      path.dirname(`icons/icon-48x48.png`)
+    )
+    const secondIconPath = path.join(
+      `public`,
+      path.dirname(`other-icons/icon-48x48.png`)
+    )
+
+    const calls = fs.mkdirSync.mock.calls
```

Comment:
```suggestion
    expect(fs.mkdirSync).toHaveBeenNthCalledWith(1, firstIconPath)
    expect(fs.mkdirSync).toHaveBeenNthCalledWith(2, secondIconPath)
```

I haven't tested this--but just a suggestion!

Reply:
I had it like that originally, however I modified it to use calls to be inline with the other tests in the file. I can change it back if you'd like

Reply:
@me4502 My 2 cents: calls are probably the less clean way to do this (I wrote some of these tests...I just didn't no this other option existed). This seems much cleaner and robust. Do as you wish imo. Another PR to clean up all the tests is welcome.

Reply:
If it'd make more sense to clean up all tests in a single PR, I can do that instead.

Reply:
It's a fair response to continue with what was _in_ the test/file to begin wit though--so don't want to dissuade from that behavior.

Let's revisit this in a future PR! For now: 🚀

Reply:
:man_shrugging: I have no preference. See if @DSchau does. My guess is he won't.